microk8s-resources/actions/common/utils.sh

```diff
@@ -219,6 +219,88 @@ skip_opt_in_config() {
     fi
 }
 
+handle_args() {
+  local args=("$@")
+  for arg in "${args[@]}"; do
+    echo "Removing argument: $arg from $service_name"
+    skip_opt_in_local_config "$arg" "$service_name"
```

Comment:
Where do we get this `$service_name` from?

Reply:
Fixed.